src/diffusers/models/unet_2d_blocks.py

```diff
@@ -1310,6 +1324,7 @@ def __init__(
                     norm_num_groups=resnet_groups,
                     bias=True,
                     upcast_softmax=True,
+                    processor=CrossAttnAddedKVProcessor(),
```

Comment:
Would this be replaced by `CrossAttnProcessor` if the user disables xformers?

Reply:
Yes, but it doesn't really work at the moment for UnCLIP anyways.

Reply:
Will throw an error message for now